drivers/ata8510/include/ata8510_params.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2016 Antonio Galea <antonio.galea@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ata8510
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for the ATA8510 driver
+ *
+ * @author      Antonio Galea <antonio.galea@gmail.com>
+ * @author      Roberto Asquini <bobasquins@gmail.com>
+ */
+
+#ifndef ATA8510_PARAMS_H
+#define ATA8510_PARAMS_H
+
+#include "board.h"
+#include "ata8510.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Set default configuration parameters for the ATA8510 driver
+ * @{
+ */
+#ifndef ATA8510_PARAM_SPI
+#define ATA8510_PARAM_SPI         SPI_DEV(0)
+#endif
+#ifndef ATA8510_PARAM_SPI_CLK
+#define ATA8510_PARAM_SPI_CLK     (SPI_CLK_100KHZ)
```

Comment:
Why so slow?

Reply:
i can reliably use the radio at 200kHz. In the past I did some attempts at pushing it more, but failed. My previous driver was written very poorly, though, the present might work better.